component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPUtils.java

```diff
@@ -540,4 +540,17 @@ public static boolean isLocalUser(AuthenticationContext context) {
         }
         return false;
     }
+
```

Comment:
additional new line

Reply:
addressed with [31f8d49](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/136/commits/31f8d49f284a10796db42765fa50021f9a6ebd07) in [https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/136](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/136)